config/config.go

```diff
@@ -66,25 +67,25 @@ type Configuration struct {
 // GetDefaultConfig returns the default configration options
 func GetDefaultConfig() Configuration {
 	return Configuration{
-		DB:                 DefaultDB,
-		DBURL:              GetDBURL(),
-		Trace:              false,
-		DBLogMode:          true,
-		EthRPC:             DefaultEthRPC,
-		TxsPerBatch:        2,
-		PollingInterval:    DefaultPollingInterval,
-		ServerPort:         DefaultSeverPort,
-		ConfirmationBlocks: DefaultConfirmationBlocks,
-
-		RollupAddress:   ethCmn.Address{}.String(),
-		BurnAuction:     ethCmn.Address{}.String(),
-		TokenRegistry:   ethCmn.Address{}.String(),
-		AccountRegistry: ethCmn.Address{}.String(),
-		DepositManager:  ethCmn.Address{}.String(),
-		State:           ethCmn.Address{}.String(),
-		Transfer:        ethCmn.Address{}.String(),
-		MassMigration:   ethCmn.Address{}.String(),
-		Create2Transfer: ethCmn.Address{}.String(),
+		DB:                     DefaultDB,
+		DBURL:                  GetDBURL(),
+		Trace:                  false,
+		DBLogMode:              false,
+		EthRPC:                 DefaultEthRPC,
+		TxsPerCommitment:       1,
+		MaxCommitmentsPerBatch: 100,
```

Comment:
These two numbers feel not right, why we parameterize them here?
Note that the contract is now parameterized to MaxCommitmentsPerBatch=32